### PR TITLE
More storage boxes

### LINKED ISSUE #56844: Toolboxes recipes need updating.
### Is your feature request related to a problem? Please describe.

The workshop toolbox have a whopping 34.4L storage space, but currently, it is still made from 10 small metal sheet (each of which are only 0.25L), some nails, and a single bundle of wire. Not to mention it not using tools at all to make. The weight of the workshop toolbox is also **noticeably** more than the components used to craft them.
Same goes for normal sized one.  8L of storage from 5 small metal sheets, nails, and a wire with absolutely no tools, and the box itself weights more than the components.

### Solution you would like.

Something this big could use normal sheet metals to make. 

### Describe alternatives you have considered.

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mx-moonling**
#### Summary
Content "Adds and standardizes storage boxes"

#### Purpose of change
Some of the only non wearable containers are cardboard boxes. They have very low limits on how much weight they can contain, and a low max item length. The item length also doesn't correspond to their sizes as marked in the code.

Toolboxes and cardboard boxes require nonsensical materials and tools.

Fixes #56844

#### Describe the solution
Adds two new materials for boxes:
- wood: very bulky walls, easy to find the tools to make, 5 times the weight capacity of cardboard. 
- metal: thin walls, medium and above require arc wielder, 10 times the weight capacity of cardboard. 

Standardizes the box sizes behind the scenes, to make material requirements more sensible and easily scalable if necessary. 
Adds a new box size, oversize, to hold longer items and panels
- small box:  7 x 15 x 20 = 2 100 cm3
- medium box:  25 x 40 x 50 = 50 000 cm3
- large box:  60 x 60 x 90 = 324 000 cm3
- oversize box: 200 x 50 x 20 = 200 000 cm3

Doubles the maximum weight capacity for a small cardboard box.

Makes the material components and the skill used for toolboxes more sensible. Details in document below.

#### Describe alternatives you've considered
The max length that a box can contain is based on its longest side. I set out my arguments why I think this is best in draft PR #56874.

The max length could depend on the diagonal of a side, or the whole shape.

#### Testing
Opened a debug world with minimal mods, crafted and deconstructed boxes, smashed placed large cardboard boxes.

#### Additional context
The interior volume is the same regardless of materials, the outside volume depends on what it is made of.

I consider that the large and oversize boxes needs double thickness walls.

### CODE REVIEW

**Comment by PatrikLundell** [comment]
Have you verified that the oversize box can fit into any vehicle storage? A box that can only be transported by carrying it is of limited use (try moving lots of mattresses, and you'll see what I mean).

File: data/json/items/containers.json [hunk 1]
```diff
@@ -689,10 +665,10 @@
     "name": { "str": "large cardboard box", "str_pl": "large cardboard boxes" },
     "looks_like": "box_medium",
     "description": "A very large cardboard box, the sort children would have loved to hide in, when there were still children.",
-    "weight": "1250 g",
-    "volume": "325000 ml",
-    "longest_side": "90 cm",
-    "//": "A box that children could hide in should be big enough to cram a (un)dead child or uncomfortable adult in (90x90x60)",
+    "weight": "5 kg",
```
Comment (bombasticSlacks) [inline data/json/items/containers.json:668]: These weights seem really high to me. Where did you get these values, boxes I'm seeing on Uhaul aren't close to this weight.

Comment (mx-moonling) [reply data/json/items/containers.json:668]: I got the weights from taking the medium cardboard box, assuming the resources used to craft was more or less correct, then scaling up or down from there, based on the relative size and area. 

I should have checked for cardboard boxes more carefully, as they are quite standardized. 

https://www.uhaul.com/MovingSupplies/Boxes/Double-Wall-Boxes/Heavy-Duty-Extra-Large-Moving-Box-Double-Walled-24-x-18-x-24 
This is a slightly smaller size to the large cardboard box, and weighs slightly under 2 kg, so I will scale by a factor of 2.5.

**Comment by mx-moonling** [comment]
[containers.xlsx](https://github.com/CleverRaven/Cataclysm-DDA/files/8516731/containers.xlsx)

This document has the excel spreadsheet I used to calculate the adjusted volume and weight of the containers. This may be useful in the future if you want to add different sized boxes, or a different material, but can mostly be disregarded.

**Comment by mx-moonling** [comment]
> Have you verified that the oversize box can fit into any vehicle storage? A box that can only be transported by carrying it is of limited use (try moving lots of mattresses, and you'll see what I mean).

I checked in-game, and yes. Oversize boxes are smaller than large boxes, and vehicle storage does not have have a maximum length for items. You can even fit two oversize boxes into one cargo space! Apart from wooden oversize boxes, which are very bulky.

**Comment by bombasticSlacks** [comment]
As long as the basis starts in reality I'm good with it, it was just the one entry in your spreadsheet with no explanation.

**Comment by mx-moonling** [comment]
https://www.uhaul.com/MovingSupplies/Boxes/Double-Wall-Boxes/Heavy-Duty-Extra-Large-Moving-Box-Double-Walled-24-x-18-x-24
This box is 60 x 60 x 45 and weighs 1.8 kg. It is double walled. 

The large cardboard box (90 x 60 x 60) is quite close in overall area, and is also double walled. It weighs 2 kg.

https://www.uhaul.com/MovingSupplies/Boxes/Standard-Sized-Moving-Boxes/Small-Moving-Box
This box is roughly 40 x 30 x 30, and weighs 0.45 kg.

The medium cardboard box (50 x 40 x 25) is again quite similar in overall area, and weighs 0.4 kg.

The small cardboard box is harder to pin to a standardized box like from U-Haul. The weight was already determined based on the weight of the other boxes, and their relative areas.

If you wish to check the weights of the boxes linked, check in the FAQ on the item.

My research does make me believe that the boxes are a little sturdier than they might be, but not by a significant factor. Personally I think that reducing their strength could cause problems if they're already being used for storage, would reduce their already limited utility, and would be outside the purview of this PR.

**Pull request merged**